VRCXDiscordTracker/Program.cs

```diff
@@ -112,10 +107,50 @@ public static void OnException(Exception e, string exceptionType)
         {
             Process.Start(new ProcessStartInfo()
             {
-                FileName = "https://github.com/tomacheese/" + AppConstants.AppName + "/issues/new?body=" + Uri.EscapeDataString(errorDetailAndStacktrace),
+                FileName = $"https://github.com/tomacheese/{AppConstants.AppName}/issues/new?body={Uri.EscapeDataString(GetErrorDetails(e, true))}",
+
                 UseShellExecute = true,
             });
         }
         Application.Exit();
     }
+
+    private static string GetErrorDetails(Exception e, bool isMarkdown)
+    {
+        var sb = new StringBuilder();
+        var fence = isMarkdown ? "```" : string.Empty;
+
+        void AppendSection(string title, string content)
+        {
+            if (isMarkdown)
+            {
+                sb.AppendLine(CultureInfo.InvariantCulture, $"## {title}\n\n{fence}\n{content}\n{fence}\n");
+            }
+            else
+            {
+                sb.AppendLine(CultureInfo.InvariantCulture, $"----- {title} -----\n{content}\n");
+            }
+        }
+
+        Exception? current = e;
+        var level = 0;
+        while (current != null)
+        {
+            var title = level == 0
+                ? "Error"
+                : $"Inner Exception (Level {level})";
+            AppendSection(title, $"{(current.Message ?? "<no message>")}\n{(current.StackTrace ?? "<no trace>")}");
+
+            current = current.InnerException;
+            level++;
+        }
+
+        // Environment info
+        AppendSection("Environment",
+            $"OS: {Environment.OSVersion}\n" +
+            $"CLR: {Environment.Version}\n" +
+            $"App: {AppConstants.AppName} {AppConstants.AppVersion}");
```

Comment:
一個指摘漏れてた…
AppVersionじゃなくてAppVersionStringかな

```suggestion
            $"App: {AppConstants.AppName} {AppConstants.AppVersionString}");
```